src/Commands/InstallCommand.php

```diff
@@ -43,18 +44,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln('   <href=https://spatie.be/docs/ray/v1/usage/framework-agnostic-php-project>https://spatie.be/docs/ray/v1/usage/framework-agnostic-php-project</>');
         $output->writeln('');
 
-        $helper = $this->getHelper('question');
         $question = new ConfirmationQuestion('   🤙 Do you wish to install Ray globally? (Y/n) > ', true);
 
-        if (! $helper->ask($input, $output, $question)) {
+        if (! $this->getHelper('question')->ask($input, $output, $question)) {
             $output->writeln('   Cancelling...');
 
-            return Command::SUCCESS;
+            return 0;
```

Comment:
I updated this due to the `Command::SUCCESS` constant only being available on symfony/console >= 5.4.